utils/src/ooinstall/openshift_ansible.py

```diff
@@ -11,30 +11,46 @@
 
 CFG = None
 
+ROLES_TO_GROUPS_MAP = {
+    'master': 'masters',
+    'node': 'nodes',
+    'storage': 'nfs'
+}
+
+VARIABLES_MAP = {
+    'ansible_ssh_user':'ansible_ssh_user',
+    'ansible_config':'ansible_config',
+    'ansible_log_path':'ansible_log_path',
```

Comment:
Mostly I was putting them there to indicate that they are "mapped variables" and thus supported. This would let us later know if they passed in unmapped variables and we could print messages or trigger other behavior based on that.

But I'm not overly attached to the idea.